### PR TITLE
Update protobuf specification (new structure)

### PR DESCRIPTION
**Pull request by StephanieBracaloni**
* Single package: iotics.api
* Includes coding language options when available
* Merge service and models (Request/Response types are in the same file as their corresponding service)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Iotics API
+as their corresponding service# Iotics API
 
 Create Data Mesh. Use interoperable digital twins to create data interactions and build powerful real-time data products. This repository contains protocol buffers specification for Iotics API.
```
Comment (smartrics) [inline README.md:3]: can you pls remove "Create Data Mesh". 

also

"this repository contains protocol buffers and gRPC service definitions for the Iotics API"

**Review by smartrics — COMMENTED** [review]
do we need the buf.lock file checked in which seems to be generated?

**Review by smartrics — APPROVED** [review]
fine by me - I would like the readme text changed but Phil will  have the last work on it.

**Review by philbelfast — APPROVED** [review]
Reference to Data Mesh can be removed later if agreed with PMs.

**Comment by StephanieBracaloni** [comment]
> fine by me - I would like the readme text changed but Phil will have the last work on it.

I would be happy to remove any reference to buf, I think we should just expose the spec without adding any opinionated linter/generator/tool.
If we are keeping the buf linter we should keep the lock file (it keeps track of the dependencies version/commit hash)

**Comment by mihai-dinculescu** [comment]
> > fine by me - I would like the readme text changed but Phil will have the last work on it.
> 
> I would be happy to remove any reference to buf, I think we should just expose the spec without adding any opinionated linter/generator/tool. If we are keeping the buf linter we should keep the lock file (it keeps track of the dependencies version/commit hash)

I think there's no harm in keeping them. If someone doesn't need/want buf, they can just ignore it and focus on the proto files.

Google does the same. See https://github.com/grpc-ecosystem/grpc-gateway

**Review by mihai-dinculescu — APPROVED** [review]

**Pull request merged**